frontend-react/src/styles/_uswds_overrides.scss

```diff
@@ -23,89 +23,89 @@ i.e.
 ----------------------------------------
 */
 
-html, body {
-  height: 100%;
+html,
+body {
+    height: 100%;
 }
 
 .footer {
-  flex-shrink: 0;
+    flex-shrink: 0;
 }
 
 body {
-  display: flex;
-  min-height: 100vh;
-  flex-direction: column;
+    display: flex;
+    min-height: 100vh;
+    flex-direction: column;
 }
 
 main {
-  flex: 1 0 auto;
-  width: 100%;
-  min-height: calc(100vh - 230px);
-  padding: 0 0 4rem 0;
-  border-top: 1px solid #dfe1e2;
+    flex: 1 0 auto;
+    width: 100%;
+    min-height: calc(100vh - 230px);
+    padding: 0 0 4rem 0;
+    border-top: 1px solid #dfe1e2;
 }
 
-.usa-banner__inner, .usa-banner__header--expanded .usa-banner__inner {
-  margin-left: 0;
-  margin-right: 0;
+.usa-banner__inner,
+.usa-banner__header--expanded .usa-banner__inner {
+    margin-left: 0;
+    margin-right: 0;
 }
 
 // NAVIGATION
 
 .usa-overlay {
-  z-index: 300;
+    z-index: 300;
 }
 
 .usa-nav__close {
-  cursor: pointer;
+    cursor: pointer;
 }
 
 .usa-nav__primary {
-  margin-top: 0;
-  margin-bottom: 0;
-  list-style: none;
+    margin-top: 0;
+    margin-bottom: 0;
+    list-style: none;
 }
 
 .usa-logo {
-  margin: units(3) units(1.5) units(3) 0;
+    margin: units(3) units(1.5) units(3) 0;
 }
 
 .usa-nav__request-account {
-  font-weight: normal;
-  text-decoration: underline;
+    font-weight: normal;
+    text-decoration: underline;
 }
 
 .usa-nav__primary-item a.usa-button {
-  color: color('primary');
+    color: color("primary");
 }
 
 // Breadcrumb
 
 .usa-breadcrumb {
-  margin-top: units(2);
+    margin-top: units(2);
 }
 
 // TYPOGRAPHY
 
 .usa-prose {
-
     h1 {
-      line-height: lh('sans', 2);
+        line-height: lh("sans", 2);
     }
 }
 
 .usa-intro {
-  line-height: units(4);
+    line-height: units(4);
 }
 
 // ALERT
-
+/*
```

Comment:
Was this intentional? Can we remove it?

Reply:
Thanks for pointing this out. I was probably experimenting and trying to understand why this override exists as it may have been interfering with my content. Will look into this.